doc/source/installation/apt_repo.rst

```diff
@@ -17,8 +17,8 @@ Supported Releases:
 
  * 12.04LTS (precise)
  * 14.04LTS (trusty)
- * 15.10 (wily)
```

Comment:
This could cause unnecessary confusion, because different versions were supported at different time. This document lists the currently supported version and installation procedure.